roles/ceph-common/tasks/misc/system_tuning.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: disable osd directory parsing by updatedb
+  command: updatedb -e /var/lib/ceph
+  changed_when: false
+  failed_when: false
+
+- name: disable transparent hugepage
+  command: "echo never > /sys/kernel/mm/transparent_hugepage/enabled"
+  changed_when: false
+  failed_when: false
+  when: disable_transparent_hugepage
```

Comment:
This should be done in a persistent way by making use of /etc/sysctl.d/ and a file for ceph. Then we can use a handler to apply the systemctl settings. See https://github.com/blueboxgroup/ursula/blob/master/roles/common/tasks/kernel-tuning.yml#L18-L23 and https://github.com/blueboxgroup/ursula/blob/master/roles/common/handlers/main.yml#L8-L9 (although it'd be nice if we could get the systmectl stuff to apply without catting files...)